### PR TITLE
doc: Document hierarchical views for the styles panel

### PR DESCRIPTION
**Pull request by cryptomilk**
Document hierarchical views for the styles panel.

### CODE REVIEW

File: doc/usermanual/lighttable/panels/styles.xml [hunk 1]
```diff
@@ -74,6 +74,11 @@
         history stack items which you want to not be part of the newly created style.
       </para>
 
+      <para>
+          The panel supports a hierarchical view, this means you can create categories using a
+          | as a sperator. For example "print|tone curve +0.5 EV" will create a print category.
```
Comment (TurboGit) [inline doc/usermanual/lighttable/panels/styles.xml]: typo: 'sperator', I propose:

...using the pipe symbol <quote>|</quote> as a separator. For example <quote>...</quote> will create...

Comment (cryptomilk) [reply doc/usermanual/lighttable/panels/styles.xml]: Fixed, thanks!

File: doc/usermanual/lighttable/panels/styles.xml [hunk 2]
```diff
@@ -87,6 +93,10 @@
         overwriting the existing one; you need to provide a new unique style name in this case.
       </para>
 
+      <para>
+        See <i>create</i> how to create a hierarchical view.
```
Comment (TurboGit) [inline doc/usermanual/lighttable/panels/styles.xml]: And \<i\> is not supported, you need to use \<emphasis\>.

Comment (TurboGit) [reply doc/usermanual/lighttable/panels/styles.xml]: This should fix the travis check.

Comment (cryptomilk) [reply doc/usermanual/lighttable/panels/styles.xml]: Fixed, lets see.

**Comment by TurboGit** [comment]
All good now! Thanks.

**Pull request merged**